.release-notes/81.md

```diff
@@ -0,0 +1,62 @@
+## Modify finished API remove assumption all responses are one-shot
+
+NOTE: This _*is*_ a breaking change.
+
+The existing Handler API made the assumption that all
+responses were one-shot, resulting in the entire response
+needing to be in memory at one time.
```

Comment:
there's context here that you don't provide that you should.

I think a good thing here would be to give an actual example.